src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.cpp

```diff
@@ -226,17 +503,86 @@ void SchwarzschildVariables<DataType>::operator()(
 template <typename DataType>
 void SchwarzschildVariables<DataType>::operator()(
     const gsl::not_null<tnsr::I<DataType, 3>*> shift_excess,
-    const gsl::not_null<Cache*> /*cache*/,
-    Tags::ShiftExcess<DataType, 3, Frame::Inertial> /*meta*/) const {
-  std::fill(shift_excess->begin(), shift_excess->end(), 0.);
+    const gsl::not_null<Cache*> cache,
+    Xcts::Tags::ShiftExcess<DataType, 3, Frame::Inertial> /*meta*/) const {
+  switch (coordinate_system) {
+    case SchwarzschildCoordinates::Isotropic: {
+      std::fill(shift_excess->begin(), shift_excess->end(), 0.);
+      break;
+    }
+    case SchwarzschildCoordinates::PainleveGullstrand: {
+      const auto& r =
+          get(cache->get_var(*this, detail::Tags::Radius<DataType>{}));
+      const DataType isotropic_prefactor = sqrt(2. * mass) / pow(r, 1.5);
+      *shift_excess = x;
+      get<0>(*shift_excess) *= isotropic_prefactor;
+      get<1>(*shift_excess) *= isotropic_prefactor;
+      get<2>(*shift_excess) *= isotropic_prefactor;
+      break;
+    }
+    case SchwarzschildCoordinates::KerrSchildIsotropic: {
+      const auto& r =
+          get(cache->get_var(*this, detail::Tags::ArealRadius<DataType>{}));
+      const auto& lapse =
+          get(cache->get_var(*this, gr::Tags::Lapse<DataType>{}));
+      const DataType isotropic_prefactor = 2. * mass * lapse / square(r);
```

Comment:
The documentation says lapse^2.  Which is correct?

Reply:
No square. It's Eq. (19) in the docs, at the bottom of the KerrSchildIsotropic docs, which is correct I think.

Reply:
You are correct.  I was reading the wrong equation.